alibi/explainers/ale.py

```diff
@@ -244,7 +265,8 @@ def bisect_fun(fun: Callable, target: float, lo: int, hi: int) -> int:
 
     Returns
     -------
-    Integer index.
+    lo : int
```

Comment:
Possible fix for the italic/normal font issue in return.

Reply:
I would avoid repeating the type return information as that is already given by the type hint and is parsed into the static docs. Repeating this manually risks inconsistencies as time goes by. I would rather stick with slightly inconsistent formatting than risk type information being out of sync.